### ISSUE TITLE
with more reducers, state has data for all app, no only for locale

### ISSUE THREAD

**Issue by burrag** [ryandrewjohnson/react-localize-redux#8]

**Comment by ryandrewjohnson** [comment]
If you are adding `localeReducer` with `combineReducers` under a specific prop you will need to ensure that you use that slice of the state when passing to selectors instead of passing in entire state.

```javascript
// so in your case do this instead of passing in entire state...
const mapStateToProps = state => ({
  translate: getTranslate(state.locale)
});
```\
This goes for any of the selectors that show taking `state` as a param.

There was an issue with the `localize` function when trying to use it when `localeReducer` was added using `combineReducers`. I have pushed a fix for this in v2.1.0 where you can now pass an additional parameter to `localize` called `slice` which should be the prop you used for `localeReducer` in `combineReducers`.

```javascript
// so lets say you use locale as the prop...
const store = createStore(combineReducers({
  locale: localeReducer
}));

// you now need to pass that to localize
export default localize(Component, 'locale');
```

Please let me know if this latest release fixes this issue.

**Comment by burrag** [comment]
it was fixed, thanks
